department-of-veterans-affairs/va.gov-cms
# VACMS-4541: Prevent non admin user from editing administration.

## Description
See #4541

## Testing done
Visual / Behat

## QA steps
- As an administrator, add Content publisher and Content creator - Vet Center roles to `vanessa.luxen@va.gov` account
- Add Create perms on main perms page for Vet Center Locations List, Vet Center Outstation and Vet Center Mobile Vet Center content types for Content creator - Vet Center role
- Assign `vanessa.luxen@va.gov` account all Vet Center sections.
- Add a VC Cap, and visually verify that all vc's are available in Owner dropdown
- Edit an existing Vet Center Outstation, and visually verify that the Owner dropdown is disabled.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
